src/components/Queue.vue

```diff
@@ -1,15 +1,24 @@
 <template id="queue">
-
-    <div class="content narrow">
+    <transition name="fade"  v-if="currentlyShown">
+    <div class="content narrow queuedVouchers">
 
         <h1 v-on:click="collapsed = !collapsed" class="expandable queue" v-bind:class="{'expanded' : !collapsed}">Queued vouchers</h1>
 
         <transition name="fade" v-if="show">
-            <div class="goodmessage queue">
-                [xXx] Thanks! We can't confirm the <strong>{{ vouchers.length }}</strong> vouchers just now, there may be a network problem. Keep adding to this list until they all go through.
+            <div v-if="!message" class="goodmessage queue">
+                Thanks! We can't confirm the <strong>{{ vouchers.length }}</strong> vouchers just now, there may be a network problem. When you have a better signal, click 'Submit queued vouchers' to try and send them again.
+            </div>
```

Comment:
I think this message should just be "You have x vouchers in your queue". The current message above would be confusing to see if you were viewing your queue containing vouchers that had been added to it during an earlier session. If you want I can do a PR for this after you merge in.

I also think the 'Voucher added to your queue' message should be revised to indicate there being a lack of signal - I'll add this to my PR too (and will make a card for it).